scripts/pin-to-cluster.sh

```diff
@@ -66,6 +66,15 @@ root_cid=$(ipfs-cluster-ctl \
 
 preview_url="https://$root_cid.ipfs.dweb.link"
 
+# Attempt to fetch the CID.
+# Ignore the response, but fail on error.
+# Wait 60s for a response. Wait 10s after a failure. Try the same again, twice.
+can_get_from_gateway=$(curl ---max-time 60 --retry 2 --retry-delay 10 --retry-connrefused --fail --silent --show-error "$preview_url") || {
```

Comment:
Wouldn't it make more sense to test whether this CID is available on the cluster ?

Reply:
Yes! I was going to go that route here, but then parsing the the output of the `ipfs-cluster-ctl status` command was a little messy and had to make a bunch of assumptions about the configuration of the cluster. 

Adding the `ipfs-cluster-ctl add --wait` flag will do that job in a more robust way, and I'll add that to this PR if it lands soon, or i'll raise a new one to add it later.